src/Illuminate/Concurrency/ProcessDriver.php

```diff
@@ -27,30 +27,52 @@ public function __construct(protected ProcessFactory $processFactory)
     /**
      * Run the given tasks concurrently and return an array containing the results.
      */
-    public function run(Closure|array $tasks): array
+    public function run(Closure|array $tasks, int $timeout = null): array
```

Comment:
IN order to be forward compatible with php version >= 8.4.

For further information you can look [here](https://www.php.net/manual/en/migration84.deprecated.php#migration84.deprecated.core.implicitly-nullable-parameter)

```suggestion
    public function run(Closure|array $tasks, ?int $timeout = null): array
```